chinemelu/hotel-locator
# Add Custom Error Messages in the Application

#### What does this PR do?
- add custom error messages

#### Description of Task to be completed?
- add error message on timeout
- modify axiosCalls class to produce consistent response objects
- produce a message when there is no search result
- add error messages for a situation whereby the image response fails

#### How should this be manually tested?
- clone the folder
- cd into the folder
- run `npm run install` and `npm run serve` respectively on the terminal
- navigate to `/` on the browser
- type "anywhere" in the search result to see the message for "no-result"
- try switching off internet connection and try to search for a hotel location to get timeout error messages

#### Any background context you want to provide?
NA

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-07-31 at 11 53 46 PM" src="https://user-images.githubusercontent.com/26977462/89085836-3caa5e80-d389-11ea-8fa2-8c92e8e60a91.png">
<img width="1440" alt="Screen Shot 2020-07-31 at 11 54 19 PM" src="https://user-images.githubusercontent.com/26977462/89085992-4764f380-d389-11ea-8729-4d603f424523.png">


#### Questions:
NA
